index.d.ts

```diff
@@ -255,6 +262,61 @@ declare namespace Eris {
     permissions?: ApplicationCommandPermissions[];
   }
 
+  // Auto Moderation
+  interface AutoModerationAction {
+    metadata?: AutoModerationActionMetadata;
+    type: AutoModerationActionType;
+  }
+  interface AutoModerationActionExecution {
```

Comment:
Shouldn't this be made a class?

Reply:
Yeah probably. Or at least not make it snake_case, but a class would probably be better

Reply:
Why should it be a class? I doesn't have any specific methods and there's no other reason for this to be a class
a camel cased object would work just fine

Reply:
Audit Log Entry is a class

Reply:
Class or not, properties should be camelCase and you could turn channel_id into a channel.

Reply:
> Audit Log Entry is a class

Existing useless classes are not a reason to make more useless classes

Reply:
Was this not the exact conversion we had with the attachment class, where we ruled that it's probably better to just have it as a class?

Reply:
Attachments are used in more than one place and are used in collections